core/src/test/java/com/linecorp/armeria/internal/common/DefaultRequestTargetTest.java

```diff
@@ -143,27 +143,27 @@ void serverShouldAcceptGoodDoubleDotPatterns(String pattern) {
     void dotsAndEqualsInNameValueQuery() {
         QUERY_SEPARATORS.forEach(qs -> {
             assertThat(forServer("/?a=..=" + qs + "b=..=")).satisfies(res -> {
-                assertThat(res).isNotNull();
-                assertThat(res.query()).isEqualTo("a=..=" + qs + "b=..=");
-                assertThat(QueryParams.fromQueryString(res.query(), true)).containsExactly(
+                assertThat(res.requestTarget).isNotNull();
```

Comment:
Should we also write e2e tests to verify that `rawPath` is correctly preserved?

Reply:
sure, added a new e2e test!